rulesengine/rulesengine/settings.py

```diff
@@ -37,6 +37,8 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'django_extensions',
+    'rules',
 ]
 
 MIDDLEWARE = [
```

Comment:
Is all of this middleware necessary? might speed requests slightly if we take out whatever we don't need (e.g. csrf, message, xframeoptions, maybe common?